pkg/status/dist/templates/forwarder.tmpl

```diff
@@ -6,6 +6,14 @@ Forwarder
   {{$key}}: {{$value}}
```

Comment:
can you filter out the `DroppedOnInput` and `Dropped` keys so that they don't show up twice?